jenkins-build-pack-website/Jenkinsfile

```diff
@@ -183,7 +183,7 @@ node ()  {
 
 					def url = 'n/a'
 					if (create_cloudfront_url == true) {
-						updateCloudFrontPolicy(service, s3ProdBucket)
+						updateCloudFrontPolicy(service, s3ProdBucket, current_environment)
```

Comment:
Make the naming convention consistent.  Should follow Camel case for var names

Reply:
Also just put 'environment' instead of current_environment

Reply:
As mentioned variable is changed from "current_environment" to "environment"